MAINTAINERS.md

```diff
@@ -0,0 +1,144 @@
+# Making a Release
+
+* Create a release branch.
+
+  ```sh
+  now=`date -u +%Y-%m-%d`
+  git checkout -b release-$now
+  ```
+
+* Create the release CHANGELOGs.
+
+  ```sh
+  yarn lerna version --no-push --conventional-graduate --no-git-tag-version
+  ```
+
+  Use `--conventional-prerelease` instead of `--conventional-graduate` if you
+  just want to generate a dev release.
+
+* Commit the results.
+
+  ```sh
+  git commit -am 'chore: lerna version'
+  ```
+
+* Identify `NEWS.md` files that need to be updated.
+  Ensure `NEWS.md` captures all user-visible changes in prose that are
+  user-relevant.
+
+  ```sh
+  git grep '# Next'
+  ```
+
+  For each of these files, copy the version number and timestamp from the
+  adjacent `CHANGELOG.md` generated in the previous step.
+  For example,
+
+  ```diff
+  -# Next release
+  +# 0.5.1 (2021-08-12)
+  ```
+
+* Commit the results.
+
+  ```sh
+  git commit -am 'chore: Update release notes'
+  ```
+
+* Push the branch.
+
+  ```sh
+  git push -u origin release-$now
+  ```
+
+* Create a pull request and request a review.
+  This is an opportunity to:
+
+  - verify that the changes pass tests under continuous integration,
+  - reflect on whether the automatically chosen version numbers tell an
+    accurate story about the backward and mutual compatibility of the packages
+    you are about to publish,
+  - and to verify that all user-facing changes are noted in the NEWS.md files
+    with appropriate migration advice when necessary.
+
+* When your reviewer has approved your release, use `git rebase -i
+  origin/master` to remove the automatically generated `chore: lerna version`
+  commit.
+
+* Recreate the changelogs with the current date *and* generate tags for the new
+  versions. This is the effect of removing the `--no-git-tag-version` flag.
+
+  ```sh
+  yarn lerna version --no-push --conventional-graduate
+  ```
+
+* Commit the results.
+
+  ```sh
+  git commit -am 'chore: lerna version'
+  ```
+
+* Force push these changes back to the pull request branch.
+
+  ```sh
+  git push origin -f release-$now
+  ```
+
+* Ensure your dependency solution is fresh and rebuild all generated assets.
+
+  ```sh
+  yarn install
+  yarn build
+  ```
+
+* Go to the npm web page for the package, for example
+  [https://www.npmjs.com/package/ses](https://www.npmjs.com/package/ses).
+  ***However*** do not be alarmed if it does not show your new version for
+  a distressingly long time. Instead you can verify the version with
+
+  ```
+  $ npm view ses
+  ```
+
```

Comment:
It seems that this should be after the following step.  Otherwise, what's the reason to check the state of NPM before publishing the packages?